modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,35 +36,151 @@ Showcase apps demonstrate many features of {product-name}. To see these features
 +
 For more information about features, see xref:identity-management.adoc[{keycloak-service} Service]
 
-
 . To demonstrate {unifiedpush-service}, xref:push-notifications.adoc#provisioning-push-notifications-service[Provision the {unifiedpush-service} service]. For more information about features, see xref:push-notifications.adoc[{unifiedpush-service} Service]
 
-. To demonstrate {sync-service}:
-+
-NOTE: You can only demonstrate {sync-service} on Cordova.
+== Demonstrate the {sync-service}
 
-.. Provision a PostgreSQL database if there isn't one available.
-... Login to the OpenShift web console, create a new project or use an existing one
-... Click on the `Add to Project` dropdown on the top right, and choose `Browse Catalog`
-... Select `Databases` -> `Postgres` -> `PostgreSQL`, and complete the wizard to start the deployment of the database. Take notes of the values for `PostgreSQL Connection Username`, `PostgreSQL Connection Password` and `PostgreSQL Database Name`. You will need them later.
-.. Provision a new {sync-service} app from the catalog as described in xref:data-sync.adoc#sync-server-openshift[Running Data Sync in OpenShift] with the following parameters:
+. Provision a PostgreSQL database if there is none available.
```

Comment:
This stuff is outdated and guides for apb. We have now templates that provision postgres automatically. While this is still valid it makes no sense to improve it as instructions for GA will be different

Reply:
```suggestion
. Ensure a PostgreSQL database is available so that you can connect to it at a later stage.
```